behandlingslager/domene/src/main/java/no/nav/foreldrepenger/behandlingslager/behandling/familiehendelse/FamilieHendelseGrunnlagEntitet.java

```diff
@@ -253,14 +253,20 @@ public Optional<FamilieHendelseEntitet> getGjeldendeBekreftetVersjon() {
      */
     public LocalDate finnGjeldendeFødselsdato() {
         final var bekreftetVersjon = getGjeldendeBekreftetVersjon();
-        if (!bekreftetVersjon.map(FamilieHendelseEntitet::getBarna).orElse(Collections.emptyList()).isEmpty()) {
-            return bekreftetVersjon.get().getBarna().stream().map(UidentifisertBarn::getFødselsdato).findFirst().get();
+
+        if (bekreftetVersjon.isPresent()) {
```

Comment:
Er det ikke mer fluent med 
`var bekreftetBarna = bekreftetVersjon.map(FamilieHendelseEntitet::getBarna).orElse(List.of())
if (!bekreftetBarna.isEmpty()) {
   return bekreftetBarna.stream().map(UidentifisertBarn::getFødselsdato).findFirst().get() / orElseThrow
}

....
`

Reply:
FamilieHendelseEntitet er noe rotete - den har både getBarna() og getFødselsdato() hvor jeg syns getFødselsdato har ingenting i FamilieHendelse å gjøre. 

Prøvde å være bare noe mer ryddig her - hente bekreftet - hente barna - hente fødselsdato fra `UidentifisertBarn` og ikke fra `FamilieHendelseEntitet`.

Reply:
getFødselsdato() er ekstern utility så man ikke traverserer getBarna overalt.
Men den du fjernet sjekket da vitterlig på bekreftetVersjon.map(FamilieHendelseEntitet::getBarna)

Reply:
Da endret jeg etter din forslag.